public/main.js

```diff
@@ -1,5 +1,38 @@
 const qrButton = document.getElementById('createqr')
 
+window.onload = qreator()
```

Comment:
your calling this function when the page loads, you want to call it once `onload` fires:

```js
window.onload = () => {
  qreator()
}
```